src/components/Indicator/Indicator.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { VibeComponent, VibeComponentProps } from "../../types";
+import styles from "./Indicator.module.scss";
+import { getStyle } from "../../helpers/typesciptCssModulesHelper";
+import { camelCase } from "lodash-es";
+import { IndicatorColor } from "./IndicatorConstants";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import { useMergeRefs } from "../../hooks";
+
+export interface IndicatorProps extends VibeComponentProps {
+  indicatorColor?: IndicatorColor;
+  ariaLabel?: string;
```

Comment:
sure is needed? worth to see how people approach it in different places/ ask Talya